### DISCUSSION TITLE
中文版《Jekyll建站完全指南》发布

### DISCUSSION THREAD

**Discussion by GuangLun2000** [GuangLun2000/GuangLun2000.github.io#1] [Announcements]
欢迎大家提出意见/建议

URL：https://caihanlin.com/blogs/web/

**Comment by GuangLun2000** [comment]
欢迎讨论！

**Reply by jayzhou1979** [reply]
支持

**Comment by xll0328** [comment]
我想更改背景如何更改啊（改颜色，如果可以的背景编程某张图片然后可以调透明色，所有页面都统一）

**Reply by GuangLun2000** [reply]
Hi, 目前还没有提供背景色更换的功能，需要修改的话，工作量不小，很多地方都需要重构🥹

**Reply by GuangLun2000** [reply]
也许可以考虑这个项目：https://github.com/HanlinCAI/HanlinCAI.github.io

**Comment by xll0328** [comment]
谢谢你🙆我在CSS文件里加了background后成功啦
---- 回复的原邮件 ----
发件人 Hanlin Cai ***@***.***> 发送日期 2023年07月12日 20:59 收件人 ***@***.***> 抄送人 Songning Lai ***@***.***>,
Comment ***@***.***> 主题 Re: [GuangLun2000/GuangLun2000.github.io] 中文版《Jekyll建站完全指南》发布 (Discussion #1)  

Hi, 目前还没有提供背景色更换的功能，需要修改的话，工作量不小，很多地方都需要重构🥹

**Reply by GuangLun2000** [reply]
那样的话太好啦～谢谢你采用这个仓库

**Comment by matthewfan0227** [comment]
请问搭建完成后如何插入可以查看访客记录的插件？

**Reply by GuangLun2000** [reply]
可以使用这个：
https://counter.dev/dashboard.html#
